src/os/ui/column/mapping/columnmappingform.js

```diff
@@ -215,7 +213,7 @@ os.ui.column.mapping.ColumnMappingFormCtrl.prototype.validateLayers_ = function(
   }
 
   var incompleteLayer = ol.array.find(columns, function(item) {
-    return (!item.layer || item.layer.length == 0 || !item.column || item.column.length == 0);
+    return (!item['layer'] || item['layer'].length == 0 || !item['column'] || item['column'].length == 0);
```

Comment:
Why did you do this? `item` is a `os.column.ColumnModel`, which is a typedef defined in `os.column.IColumnMapping` and should not require bracket notation.

Reply:
This solution was to fix the first screenshot. When it wasn't in bracket notation the  incompleteLayer would always be set to false after building it. I assume closure didnt like something.

Reply:
Hmm. If `os.column.ColumnModel` is being displayed in an Angular template, that might explain it. However, we should track that down. If the names need to be preserved, we can move the typedefs to externs rather than switching to bracket notation and that will prevent their minification.

Reply:
Other parts of the file use bracket notation for the column model too
If we change the typedef to an extern, these will need to be updated
https://github.com/ngageoint/opensphere/blob/master/src/os/ui/column/mapping/columnmappingform.js#L147
https://github.com/ngageoint/opensphere/blob/master/src/os/ui/column/mapping/columnmappingform.js#L203
https://github.com/ngageoint/opensphere/blob/master/src/os/ui/column/mapping/columnmappingform.js#L211
https://github.com/ngageoint/opensphere/blob/master/src/os/ui/column/mapping/columnmappingform.js#L259

Reply:
@wallw-bits I updated this to so the typedef is now an extern. Is there anything else I should look at or can this change be closed?

Reply:
If the type definition is in the externs then you should no longer need the bracket notation. Once that is fixed here and in the places Doug mentioned, then I'll have a final look at it.

Reply:
Looks good